test/spec/modules/33acrossIdSystem_spec.js

```diff
@@ -412,5 +448,21 @@ describe('33acrossIdSystem', () => {
         }
       });
     });
+
+    it('returns the stored additional IDs as well', () => {
+      sinon.stub(storage, 'getDataFromLocalStorage').returns('{"eids":{"foosourcename":{"id":"someId","ext":{"stype":"ppuid"}}}}');
```

Comment:
Can we selectively listen on storage.getDataFromLocalStorage only for key "33acrossId_ext"?